setup.cfg

```diff
@@ -29,7 +29,7 @@ install_requires =
     Mopidy >= 3.0.0
     Pykka >= 2.0.1
     setuptools
-    pyalsaaudio
+    pyalsaaudio >= 0.8.0
```

Comment:
Even Debian oldstable has 0.8.4, so I think this requirement is entirely OK. :+1: